hub/auth/login.go

```diff
@@ -98,3 +98,20 @@ func Login(
 
 	return currentSession, nil
 }
+
+func IsLoggedIn(currentSession *sessionstore.HubSession) bool {
```

Comment:
we dont check here if the user is logged in, but rather if they have login creds.
maybe we rename it to something like `HasLoginCreds` or something similar